page/page.njk

```diff
@@ -48,25 +49,20 @@
 {% endblock %}
 
 {% block header %}
-  {% if isAuthenticated %}
-    {{govukHeader({
+    {% if isAuthenticated %}
+        {{govukOneLoginServiceHeader({
             serviceName: "Claim criminal injuries compensation",
-            serviceUrl: "https://www.gov.uk/claim-compensation-criminal-injury/make-claim",
-            homepageUrl: "https://www.gov.uk",
-            navigationLabel: 'Account navigation',
-            menuButtonLabel: 'Account navigation',
-            navigationClasses: 'govuk-header__navigation--right',
-            navigation: [{
-                text: 'Your applications',
-                href: '/account/dashboard'
-            },{
-                text: 'Log out',
-                href: '/account/sign-out'
-            }],
-            useTudorCrown: true
+            serviceLink: "https://www.gov.uk/claim-compensation-criminal-injury/make-claim",
+            currentUrlPathname: currentUrlPathname,
+            signOutLink: "/account/sign-out",
+            navigationItems: [{
+                href: "/account/dashboard",
```

Comment:
I'm not entirely clear why this href is hard-coded but `currentUrlPathname` is passed in to the template? Could both just be hard-coded here? Or both set to the value of `currentUrlPathname` if you're concerned that we change the URL in the future?